test/Task.ml

```diff
@@ -105,4 +135,58 @@ module TestCommandExpr = struct
       )
     )
 
+let checkEnvExists ~name ~value task =
+  let bindings = Environment.Closed.bindings task.Task.env in
+  List.exists
+    (function
+      | {Environment. name = n; value = Value v; _} when name = n ->
+        if v = value
+        then true
+        else false
+      | _ -> false)
+    bindings
+
+  let%test "#{OCAMLPATH} depending on os" =
+    let dep = Package.{
+      dep with
+      exportedEnv = [
+        {
+          ExportedEnv.
+          name = "OCAMLPATH";
+          value = "#{'one' : 'two'}";
+          exclusive = false;
+          scope = Local;
+        };
+        {
+          ExportedEnv.
+          name = "PATH";
+          value = "#{'/bin' : '/usr/bin'}";
+          exclusive = false;
+          scope = Local;
+        };
+        {
+          ExportedEnv.
+          name = "OCAMLLIB";
+          value = "#{os == 'cygwin' ? ('lib' / 'ocaml') : 'lib'}";
```

Comment:
Can we do `os == 'windows'` here? This is great that we can differentiate now! 👍

Reply:
Oops, realized after looking at the other paths i was looking a test 😄

Reply:
Yes, now it's possible to do `os == 'windows'`